packages/harmony/.svgrrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  template: require('./svgr-template'),
+  dimensions: false,
+  replaceAttrValues: {
+    red: '{props.fill}',
+    '#f00': '{props.fillSecondary}'
```

Comment:
I'm not sure where is the best place but we should probably document this special behavior / requirement that the incoming SVGs have to use certain colors in order to be correctly filled.

Also, am I misreading this or do both of these values map to red? I thought it was red/black...

Reply:
And last question: Are `fill` and `secondaryFill` props on the resulting Icon component? I didn't think harmony icons supported those fields, but just used a color value.

Reply:
sorry this should also be removed for now. we can use this moving forward to pass the "fill" prop instead of "color" depending on the implementation we want to go with.